### ISSUE TITLE
'Tab' button does not work when shift button from last button

### ISSUE THREAD

**Issue by AllenMiaoFil** [dotnet/maui#7064]
### Description

The 'Tab' functionality does not work any more when you shift button with 'Tab' from the last button to first button.

And There is no focus functionality in MacOS, has created bug [here](https://github.com/dotnet/maui/issues/6482)

### Steps to Reproduce

1. Create two MAUI buttons on main page.
2. Run application. 
3. Use 'Tab' to shift button.
4. When he last button get focus, continue use 'Tab' to shift button. you will see the focus never back to first button.

### Version with bug

Release Candidate 2

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

Window 10

**Comment by kristinx0211** [comment]
repro on windows with vs 17.3.0 Preview 2.0 [32515.5.main]. repro project:
[MauiApp7.zip](https://github.com/dotnet/maui/files/8697335/MauiApp7.zip)

**Comment by rachelkang** [comment]
Hi, @AllenMiaoFil - thanks for bringing this to our attention. This behavior you describe seeing on Windows is the same as the WinUI platform behavior, and therefore is actually the current expected behavior. While `Tab` will bring you to the next element (from the first button to the second button), `Shift + Tab` will bring you back to the previous element (From the second button to the first button). 

That being said, if you have any specific scenarios to share or any further thoughts/ideas for how keyboard navigation should work, please do share, as we will be further developing on it in the future.

**Comment by AllenMiaoFil** [comment]
Hey @rachelkang For now I have no requirement for this one, just when I open application, I found 'Tab' doesn't work any more on last button, and can't use Shift + Tab when stay on first button. it's different with other applications.

**Comment by rachelkang** [comment]
@AllenMiaoFil You mention that it doesn't work "any more" - when is the last time you saw it working on .NET MAUI? And if you don't mind sharing, what other applications is this working as you expect it to? This information will help us investigate further :)

**Comment by rachelkang** [comment]
@AllenMiaoFil I've just learned that Windows has a property to change the keyboard navigation mode to reflect just what you're looking for! The property is called [`TabFocusNavigation`](https://docs.microsoft.com/uwp/api/windows.ui.xaml.uielement.tabfocusnavigation?view=winrt-22621#windows-ui-xaml-uielement-tabfocusnavigation) and can be leveraged with handlers.

You can learn more about .NET MAUI handlers here: https://docs.microsoft.com/dotnet/maui/user-interface/handlers/customize

For this scenario, putting this bit of code in your code-behind and then calling on it from the constructor will do the trick :)
```cs
void ModifyTabNavigation()
{
        Microsoft.Maui.Handlers.LayoutHandler.Mapper.AppendToMapping("MyCustomization", (handler, view) =>
        {
#if WINDOWS
            handler.PlatformView.TabFocusNavigation = Microsoft.UI.Xaml.Input.KeyboardNavigationMode.Cycle;
#endif
        });
}
```

Let me know if you have any further questions :)

**Comment by AllenMiaoFil** [comment]
Hey @rachelkang, I think you mentioned is an other question, for right now, can you repro what I mentioned with the 'Steps to Reproduce'? maybe we are not in same page.

**Comment by rachelkang** [comment]
Hi, @AllenMiaoFil - yes, I'm able to repro what you mentioned in the "Steps to Reproduce" you provided, and the code I shared helps to remedy that so you can "Tab" from the last button back to the first as you wish.

Here's an updated sample incorporating the code mentioned:
[MauiApp7.zip](https://github.com/dotnet/maui/files/9507808/MauiApp7.zip)

**Comment by jsuarezruiz** [comment]
Testing with the current version seems that the behavior on Windows is correct:
![issue-7064](https://user-images.githubusercontent.com/6755973/232805749-3a570928-893a-45c8-b131-a37039c6de8f.gif)
